libbeat/scripts/Makefile

```diff
@@ -118,7 +118,7 @@ clean:: ## @build  Cleans up all files generated by the build steps
 	# Clean index pattern
 	rm -f $(PWD)/_meta/kibana/index-pattern/${BEAT_NAME}.json
 	# Remove temp templates
-	-rm ${BEAT_NAME}.template*.json
```

Comment:
This should not be needed anymore on master as no templates are generated anymore. All are loaded dynamically from fields.yml.